HW0/P6/P6B.py

```diff
@@ -15,17 +30,18 @@ def burnTime(t):
     pool = mp.Pool(P)
 
     # Use a variety of wait times
+    wait_time = [10**-6,5*10**-6,10**-5,5*10**-5,10**-4,5*10**-4,10**-3,5*10**-3,10**-2,5*10**-2,10**-1,.5,1]
```

Comment:
^ Agreed with this, use linspace or logspace to make this cleaner and more maintainable.